lib/components/map/connected-transit-vehicle-overlay.js

```diff
@@ -102,22 +102,31 @@ function VehicleTooltip(props) {
   )
 }
 
+// Settings where caret is touching the border of the circle.
+const CaretTouchingBorder = withCaret(Circle, {
+  height: 5,
+  offset: 1.5,
+  width: 10
+})
+
 // Round vehicle symbol with arrow/caret on the border,
 // and showing the route color with a transparent effect on hover.
-const IconContainer = withRouteColorBackground(
-  withCaret(Circle, { height: 5, offset: 1.5, width: 10 }),
-  {
-    alphaHex: 'aa',
-    display: 'onhover'
-  }
-)
+const IconContainer = withRouteColorBackground(CaretTouchingBorder, {
+  alphaHex: 'aa',
+  display: 'onhover'
+})
 
 // connect to the redux store
 
 const mapStateToProps = (state) => {
   const viewedRoute = state.otp.ui.viewedRoute
   const route = state.otp.transitIndex?.routes?.[viewedRoute?.routeId]
 
+  const ConfiguredIconContainer =
+    state.otp.config?.routeViewer?.vehicleIconHighlight === false
```

Comment:
I think you can replace this with `!state.otp.config?.routeViewer?.vehicleIconHighlight`

Reply:
Careful! That will treat undefined differently than this does

Reply:
ok! Feel free to disregard!

Reply:
Good catch, but yeah this is super confusing and special ✨ Since we're setting `vehicleIconHighlight` as `true` or `false` in the config, we want to check for false specifically, and distinguish the other falsy values like `""`, `0`, `null`,  and `undefined` as *not* false